test/test.js

```diff
@@ -619,6 +619,16 @@ describe('Page', function() {
       expect(error.message).toContain('Navigation Timeout Exceeded: 1ms');
       process.removeListener('unhandledRejection', unhandledRejectionHandler);
     }));
+    it('should work when option.timeout set to 0', SX(async function() {
```

Comment:
this test will run for quite some time - which is unfortunate.
Developer's time is precious! :)

I'd rather make sure that `timeout: 0` doesn't set a zero timeout watchdog:

```js
it('should disable timeout when its set to 0', SX(async function() {
  let error = null;
  await page.goto(PREFIX + '/grid.html', {timeout: 0}).then(e => error = e);
  expect(error).toBe(null);
}));
```